crates/bevy_ecs/src/schedule/mod.rs

```diff
@@ -201,6 +201,13 @@ impl Schedule {
             stage.run(world);
         }
     }
+
+    /// All the schedule stages in insertion order
```

Comment:
```suggestion
    /// Iterates over all of schedule's stages and their labels, in execution order.
```
`stage_order` is not insertion order. Insertion order is not retained.